### PR TITLE
[fb-www] Internal bootloader changes

### PR DESCRIPTION
**Pull request by trueadm**
Release notes: none

Some internal changes to ensure `Bootloader.loadModules()` global stays as is and doesn't get destructured. Also changed the incorrect usage of `context.$Realm.generator.derive` in the mocks.

### CODE REVIEW

File: test/react/mocks/fb13.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
```
Comment (gaearon) [inline test/react/mocks/fb13.js]: Nite: Extra newline

Comment (trueadm) [reply test/react/mocks/fb13.js]: Doesn't this newline make sense?

**Comment by trueadm** [comment]
Not sure why the depcheck fails? This doesn't happen locally. @hermanventer

**Comment by hermanventer** [comment]
There seems to be a bug in depcheck. It will help if you can attach the output you get when you run flow check --profile on this branch.

**Pull request closed without merging**